scripts/create-release-tag.js

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env node
+/* eslint-disable import/no-extraneous-dependencies */
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * This script creates a release on GitHub for the Backstage repository.
+ * Given a git tag, it identifies the PR created by changesets which is responsible for creating
+ * the git tag. It then uses the PR description consisting of changelogs for packages as the
+ * release description.
+ *
+ * Example:
+ *
+ * Set GITHUB_TOKEN environment variable.
+ *
+ * (Dry Run mode, will create a DRAFT release, but will not publish it.)
+ * (Draft releases are visible to maintainers and do not notify users.)
+ * $ node scripts/get-release-description v0.4.1
```

Comment:
Wait, what's this script name? Oh this is copied from elsewhere. And isn't THAT one wrong in the first place too? :)

Reply:
Didn't even see this comment, just saw the copyright header xP